x-pack/legacy/plugins/infra/public/containers/with_kuery_autocompletion.tsx

```diff
@@ -56,21 +53,13 @@ export class WithKueryAutocompletion extends React.Component<
     maxSuggestions?: number
   ) => {
     const { indexPattern } = this.props;
-    const autocompletionProvider = getAutocompleteProvider('kuery');
-    const config = {
-      get: () => true,
-    };
+    const language = 'kuery';
+    const hasQuerySuggestions = npStart.plugins.data.autocomplete.hasQuerySuggestions(language);
```

Comment:
@Kerry350 heads up that we'll have to figure out how to handle this during our migration efforts now ... we can add the data plugin to the singleton or pass it from context if that's possible with what you've done already?

Reply:
If I understand your question correctly, checkout the services files in the [data](https://github.com/elastic/kibana/blob/master/src/plugins/data/public/services.ts) & [visualizations](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/visualizations/public/np_ready/public/services.ts
) plugins.


It's a useful way of getting access to core \ architecture team plugins without passing them around.